### ISSUE TITLE
Endpoint ID is not fully logged out

### ISSUE THREAD

**Issue by caseydavenport** [projectcalico/felix#1309]
```
[INFO][41/12] calico.felix.endpoint 277: Endpoint WloadEndpointId<eth0> deleted
```

All of the Felix endpoints show up like this, which makes it hard to tell what's going on, since all WorkloadEndpoints in k8s are interface eth0.

There are a bunch of WorkloadEndpoint identifiers that we should be logging.  Importantly, the WorkloadName, which is used in k8s as the identifier.

https://github.com/projectcalico/libcalico-go/blob/master/lib/api/workloadendpoint.go#L39

**Comment by fasaxc** [comment]
@caseydavenport I've merged a fix to the 2.0.x-series branch; do you need a new release to test with?

**Comment by caseydavenport** [comment]
@fasaxc Yes, that would be useful.

**Comment by fasaxc** [comment]
Shipped in 2.0.4
